src/main/java/org/ladocuploader/app/csv/converters/UnmarriedMinorConverter.java

```diff
@@ -0,0 +1,44 @@
+package org.ladocuploader.app.csv.converters;
+
+import com.opencsv.bean.AbstractBeanField;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.ladocuploader.app.utils.HouseholdUtilities;
+import java.time.LocalDate;
+import java.util.Map;
+
+/**
+ * This converter will take marital status and age and compute unmarried minor
+ */
+@NoArgsConstructor
+@Slf4j
+public class UnmarriedMinorConverter<T, I> extends AbstractBeanField<T, I> {
```

Comment:
uh, what

Reply:
@lkemperman-cfa ? 😅

Reply:
lol should this be renamed? this is literally the question on the ECE app :/